jwst/associations/lib/rules_level2b.py

```diff
@@ -782,7 +784,7 @@ def __init__(self, *args, **kwargs):
                 DMSAttrConstraint(
                     name='exp_type',
                     sources=['exp_type'],
-                    value='nis_wfss',
+                    value='nis_wfss|nis_extcal',
```

Comment:
I think nis_extcal needs to be removed here, or else this will match all extcals, not just valid ones.

```suggestion
                    value='nis_wfss',
```